main.dart

```diff
@@ -1,3 +1,58 @@
 void main() {
-  print('hello');
+  int firstDetail = findDetail(inputData, '*', '.');
+  int secondDetail = findDetail(inputData, ':', null);
+  print(firstDetail);
+  print(secondDetail);
+
+  int phoneCount = createPhone(firstDetail, secondDetail);
+  print('Phone created is $phoneCount');
+}
+
+const inputData = [
+  '*.**::::**..*::.*…**.*::',
+  '*.**::::**..*::.*…**.*::',
+  '*.**::::**..*::.*…**.*::',
+];
+
+int findDetail(List<String> inputData, String firstDetailPart, String? secondDetailPart) {
+  int counter = 0;
+  String firstChar;
+  String secondChar;
+
+  for (String dataPart in inputData) {
+    for (int i = 0; i < dataPart.length - 1; i++) {
+      if (secondDetailPart == null) {
+        firstChar = dataPart[i];
+        if (firstChar == firstDetailPart) {
+          counter++;
+        }
+      } else {
+        firstChar = dataPart[i];
+        secondChar = dataPart[i + 1];
+        if (firstChar == firstDetailPart && secondChar == secondDetailPart) {
+          counter++;
+        }
+      }
+    }
+  }
+  return counter;
+}
+
+int createPhone(int firstDetailCount, int secondDetailCount) {
```

Comment:
if you return count, you need to call it like `getPhoneCount` or `calculatePhoneCount`